src/SIL.XForge.Scripture/ClientApp/.eslintrc.json

```diff
@@ -35,6 +35,7 @@
         ],
         "@typescript-eslint/consistent-type-definitions": "error",
         "@typescript-eslint/dot-notation": "off",
+        "@typescript-eslint/explicit-function-return-type": "warn",
```

Comment:
Some of the warnings seem not that helpful:
![](https://user-images.githubusercontent.com/6140710/214902006-61bfd2d7-61d5-45a2-8dcf-8890ff9b6787.png)

What about turning on `allowExpressions`? That drops the lint warnings from 299 to 245.

``` suggestion
        "@typescript-eslint/explicit-function-return-type": [
          "warn",
          {
            "allowExpressions": true
          }
        ],
```

Reply:
Agreed